contracts/src/factory/facets/architect/ArchitectBase.sol

```diff
@@ -353,20 +358,34 @@ abstract contract ArchitectBase is Factory, IArchitectBase, PricingModulesBase {
       membershipSettings.feeRecipient = msg.sender;
     }
 
+    address proxyInitializer = address(
+      ImplementationStorage.layout().proxyInitializer
+    );
+
     // calculate init code
     initCode = abi.encodePacked(
       type(SpaceProxy).creationCode,
       abi.encode(
-        msg.sender,
         IManagedProxyBase.ManagedProxy({
           managerSelector: IProxyManager.getImplementation.selector,
           manager: address(this)
         }),
-        ITokenOwnableBase.TokenOwnable({
-          collection: spaceToken,
-          tokenId: spaceTokenId
-        }),
-        membershipSettings
+        proxyInitializer,
+        abi.encodeCall(
+          SpaceProxyInitializer.initialize,
+          (
+            msg.sender,
+            IManagedProxyBase.ManagedProxy({
+              managerSelector: IProxyManager.getImplementation.selector,
+              manager: address(this)
+            }),
```

Comment:
Should reuse the struct or pass `address(this)` only.

Reply:
fixed in latest push